fe/src/main/java/org/apache/doris/catalog/Catalog.java

```diff
@@ -5126,13 +5126,6 @@ public void createView(CreateViewStmt stmt) throws DdlException {
         View newView = new View(tableId, tableName, columns);
         newView.setComment(stmt.getComment());
         newView.setInlineViewDef(stmt.getInlineViewDef());
-        newView.setOriginalViewDef(stmt.getInlineViewDef());
-        try {
-            newView.init();
```

Comment:
Do we need to keep this init in order to return parse error ASAP?

Reply:
Actually, I don't know why it throws an exception here. the stmt string comes from QueryStmt.toSql()...

Reply:
Maybe there is some bug in our toSql() implementation

Reply:
Add it back